### PR TITLE
minor spelling update

### PR DESCRIPTION
**Pull request by chmouel**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1259)

### CODE REVIEW

File: docs/user-guide/petset/bootstrapping/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This example shows you how to "carry over" runtime state across Pet restart by s
...
+Applications that incrementally build state usually need strong guarantees that they will not restart for extended durations. This is tricky to achieve with containers, so instead, we will ensure that the results of previous computations are transferred to future pets. Doing so is straightforward using vanilla Persistent Volumes (which Pet Set already gives you), unless the volume mount point itself needs to be initialized for the Pet to start. This is exactly the case with "virtual machine" docker images, like those based on ubuntu or fedora. Such images embed the entier rootfs of the distro, including package managers like `apt-get` that assume a certain layout of the filesystem. Meaning:
```
Comment (devin-donnelly) [inline docs/user-guide/petset/bootstrapping/index.md]: "Such images embed the entier rootfs" -> "Such images embed the entire rootfs"

**Comment by devin-donnelly** [comment]
@chmouel , I don't see the spelling update. Have you seen my review comment?

**Comment by chmouel** [comment]
@devin-donnelly apologies this took more time that it should, corrected the error and add bunch of others (using the '[mispellings](https://pypi.python.org/pypi/misspellings)' tool)

**Comment by devin-donnelly** [comment]
This is good, but it's going to introduce roughly eight merge conflicts when we launch 1.4. Do you mind if I hold onto it for a couple days?

**Pull request merged**